src/app/editor-toolbar/editor-toolbar-save/editor-toolbar-save.component.ts

```diff
@@ -0,0 +1,24 @@
+import {
```

Comment:
Add license on top of the file like in https://github.com/zzacharo/record-editor/blob/030402323099b1789c2ebe3c023413f7ee22c6be/src/app/app.component.ts